docs/blocked-export.md

```diff
@@ -1,12 +1,23 @@
 These are export PRs which the exporter should have merged by now.
 
-Usually this is caused by failing Travis checks. Read the Travis logs to understand the failure.
+Usually this is caused by failing TaskCluster checks. Read the TaskCluster logs to understand the failure.
 
-A common cause of failure is that [when many tests are affected, Travis jobs will time out](https://github.com/web-platform-tests/wpt/issues/7660). The timeout is at around 50 minutes. When this happens, the PR has to be force merged. Link to [#7660](https://github.com/web-platform-tests/wpt/issues/7660) in a comment and ask @stephenmcgruer or @Hexcles to merge the PR.
+A common problem is the TaskCluster stability check failing, which indicates
+that a test is flaky. This is very often a problem with the test itself and will
+require help from the author, so deflaking these can take some time. To avoid
+these PRs from piling up we aim to clear them from the queue and apply a fix in
+the background.
 
-If the test is flaky, try to reproduce the flakiness locally and if necessary ask the original CL author for help.
+Follow these steps (see [#23617](https://github.com/web-platform-tests/wpt/pull/23617) for a recent example):
+- copy the "Unstable Results" from the TaskCluster log into the PR for easier
+  viewing.
+- create a crbug to record the flake, which should point to the PR and be assigned to the CL author.
+- reference the new bug in the PR.
+- force merge the PR.
 
-If the failure looks unrelated to the code changes, first rebase and force push the branch. This will both bring in any recent fixes to the problem and trigger a new Travis run. If that doesn't fix the problem and other PRs are also affected, [file a wpt infra bug](https://github.com/web-platform-tests/wpt/issues/new?labels=infra).
+Another common cause of failure is that [when many tests are affected, TaskCluster jobs will time out](https://github.com/web-platform-tests/wpt/issues/7660). The timeout is at around 50 minutes. When this happens, the PR has to be force merged. Link to [#7660](https://github.com/web-platform-tests/wpt/issues/7660) in a comment and ask @stephenmcgruer or @Hexcles to merge the PR.
```

Comment:
Drive-by since you're touching this; the timeout is now 7200 seconds, so like 120 minutes.

Reply:
Done.